packages/utils/src/lib/reports/generate-stdout-summary.ts

```diff
@@ -52,7 +52,6 @@ function reportToDetailSection(report: ScoredReport): string {
         },
         {
           text: chalk.cyanBright(audit.displayValue || `${audit.value}`),
-          width: 10,
```

Comment:
Keep `width: 10`, please. Without it, the middle column doesn't fill available space and a lot more of the audit titles don't fit. Look at difference in the snapshot - the original takes much better advantage of the available width.

![image](https://github.com/code-pushup/cli/assets/34691111/20592e5c-35f6-4a97-a07f-fae77d7d8035)